### PR TITLE
PSR-192

### PR DESCRIPTION
**Pull request by dephillipsmichael**
Added phone based sign in based on mPower 2.  This supports using the phone code token that was sent to your mobile phone, so it even works on simulator.

I moved externalID sign in over to long-holding the welcome screen video, if you still want to use that.  It only works like that for DEBUG builds.

### CODE REVIEW

File: Psorcast/Psorcast/Base.lproj/PhoneSignIn.json [hunk 1]
```diff
@@ -0,0 +1,67 @@
+{
+    "identifier":"PhoneSignIn",
+    "shouldHideActions":["skip"],
+    "progressMarkers":[],
+    "steps":[
+             {
+                "identifier":"enterPhoneNumber",
+                "type":"form",
+                "title":"Let's get started",
+                "text":"Please enter your phone number so we can see if you have an existing account.\n\nWe will not use your phone number for anything else until you have completed the Consent step of this study.",
```
Comment (DanWebster) [inline Psorcast/Psorcast/Base.lproj/PhoneSignIn.json:10]:
```suggestion
                "text":"Please enter your phone number to create and verify your account with a text message.\n\nWe will NEVER sell, rent, or lease your contact information.",
```

File: Psorcast/Psorcast/Base.lproj/PhoneSignIn.json [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+                        "buttonTitle": "Learn more",
+                        "url" : "PhoneLearnMore.html"
+                    }
+                },
+                "nextStepIdentifier": "waiting"
+             },
+             {
+                 "identifier":"waiting",
+                 "type":"instruction",
+                 "title":"Your registration link is on its way!",
```
Comment (DanWebster) [inline Psorcast/Psorcast/Base.lproj/PhoneSignIn.json:43]:
```suggestion
                 "title":"Your sign-up link is on its way!",
```

**Review by DanWebster — CHANGES_REQUESTED** [review]
The 'Learn More' link on the first screen gets you out to a page that you can't 'X' out of or return from (screenshot)
IMG_AB243D87406B-1.

I got a verify and a sign-in code at the same time (screenshot).  Is this the expected functionality?  I tried putting in the second of these, but it hung the app (screenshot)

IMG_55AA166D86E6-1
IMG_A1044528AD8F-1

Added in in-line text changes for this PR in terms of language on the screen

**Comment by dephillipsmichael** [comment]
@DanWebster the learn more link is launched by BridgeApp SDK, so I'll have to check with Shannon on that one.

**Comment by dephillipsmichael** [comment]
@DanWebster this should be fixed to include all your changes, as well as fixing the sign up / sign in flow.  I believe you will still get 2 tokens from using sign up flow, but the first one should be valid.  Sign in flow seems to only send 1 token correctly.  I will keep trying to fix the 2 tokens issue with Alx.

**Comment by DanWebster** [comment]
@dephillipsmichael I am able to sign in now (with an existing account) and the 'learn more' is dismissable with a downward swipe now too.  Will merge and create story for troubleshooting the double text once I repro after deleting my account in Bridge Study manager

**Pull request merged**